debian-bookworm/piaware-support.generate-network-config.service

```diff
@@ -3,15 +3,11 @@
 
 [Unit]
 Description=Generate network configuration
-Before=network-pre.target
-Wants=network-pre.target
-After=config-ready.target set-rfkill.service NetworkManager.service
-Wants=config-ready.target
-DefaultDependencies=no
+Before=NetworkManager.service
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/python3 /usr/lib/piaware-support/generate-network-config-bookworm.py
+ExecStart=/usr/lib/piaware-support/generate_network_config_bookworm.py
```

Comment:
You can drop `.py` extension from generate_network_config_bookworm.py file, the .service file, and the .install file